api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml

```diff
@@ -84,6 +84,11 @@ properties:
       $ref: '#/components/schemas/Volume'
     title: Volumes
     type: array
+  namespace:
+    title: Namespace
+    type: string
+    description: 'The namespace to run the Spark application. Leave this empty and Ocean Spark will use the 
+      default namespace configured for the cluster'
```

Comment:
suggestion:

"The Kubernetes namespace that Spark application should run in. The provided namespace must be present in the cluster's Spark app namespace list. If empty, Ocean Spark will use the default `spark-apps` namespace"

- Adding a note that the given namespace must in the configured app namespace list at the cluster level
- Referencing `spark-apps` directly, the default namespace is not configurable